recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
> On some place of my code I missed to capsule

Makes sense.

> Maybe that could be added as a dependency? Because it seems not needed for 1.0.7.

That is a design decision decision where `conda-forge` is different from `defaults`. We don't ship all the data b/c it is a heavy download, and not always needed. Since you clearly need it you have to install both packages (basemap core and data). Adding the data package as a dependency in the core defeats the purpose of that design.

Ideally that separation should have happen in the `basemap` code itself and not only at packaging time, but `basemap` is under maintenance mode and I doubt someone will implement that.

BTW: I suggest you try `cartopy` for mss and all your mapping needs :wink: